Mollier/SAM.Core.Mollier.UI/Query/IsEqual.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System; 
+
+namespace SAM.Core.Mollier.UI
+{
+    public static partial class Query
+    {
+        public static bool IsEqual(this IMollierProcess mollierProcess1, IMollierProcess mollierProcess2, double tolerance = Tolerance.Distance)
+        {
+            if(mollierProcess1 == null || mollierProcess2 == null)
```

Comment:
Add on the top:

if(mollierProcess1 == null && mollierProcess2 == null)
{
return true;
}